en_US/configuration/configuration.md

```diff
@@ -313,16 +313,14 @@ The **String** data type represents a sequence of characters and supports severa
 
 For example:
 
-``
-rule_xlu4 {
+`rule_xlu4 {
```

Comment:
@Meggielqk this change will make it into a single line.
this is intended to be a multiline code snippet